core/src/main/scala/com/spotify/featran/CrossingFeatureBuilder.scala

```diff
@@ -65,14 +65,14 @@ private class CrossingFeatureBuilder[F] private (private val fb: FeatureBuilder[
     extends FeatureBuilder[F] {
 
   private case class CrossValue(name: String, offset: Int, value: Double)
-  private var xEnabled = false // true if current transformer will be crossed
+  private[this] var xEnabled = false // true if current transformer will be crossed
```

Comment:
What does `[this]` scope do? So it's private within `class` & `object`?

Reply:
yes and also makes the compiler not generate accessor methods, do this avoids extra calss.